ui/de-lib/src/main/java/org/iplantc/de/client/util/OntologyUtil.java

```diff
@@ -41,6 +41,11 @@ public int compare(OntologyHierarchy o1, OntologyHierarchy o2) {
     final String UNCLASSIFIED_LABEL = "Unclassified";
     final String UNCLASSIFIED_IRI_APPEND = "_unclassified";
 
+    final String BETA_ATTR = "n2t.net/ark:/99152/h1459";
+    final String BETA_VALUE = "beta";
+    final String BETA_SUB_VALUE = "releaseStatus";
+    final String BETA_SUB_UNIT = "attr";
```

Comment:
Oh awesome, I didn't realize they were in the configs already.  Yeah, I'll use the configs for these instead.